components/edit-collective/sections/InvoicesReceipts.js

```diff
@@ -54,63 +95,189 @@ const InvoicesReceipts = ({ collective }) => {
           defaultMessage="You can customize the title (and add custom text) on automatically generated receipts for financial contributions to your Collective(s), e.g., 'donation receipt' or 'tax receipt' or a phrase appropriate for your legal entity type, language, and location. Keep this field empty to use the default title:"
         />
         {/** Un-localized on purpose, because it's not localized in the actual invoice */}
-        &nbsp;<i>Payment Receipt</i>.
+        &nbsp;<i>{defaultReceiptTitlePlaceholder}</i>.
       </P>
       {error && (
         <MessageBox type="error" fontSize="14px" withIcon mb={3}>
           {i18nGraphqlException(intl, error)}
         </MessageBox>
       )}
-      <Flex flexWrap="wrap" justifyContent="space-between" maxWidth={800}>
-        <Flex flexWrap="wrap" flexDirection="column" width="100%" maxWidth={350}>
-          <StyledInput
-            placeholder="Payment Receipt"
-            defaultValue={value}
-            onChange={e => setValue(e.target.value)}
-            width="100%"
-            maxWidth={350}
-            my={3}
-          />
-
-          <StyledTextarea
-            placeholder={intl.formatMessage(messages.extraInfoPlaceholder)}
-            defaultValue={info}
-            onChange={e => setInfo(e.target.value)}
-            width="100%"
-            height="150px"
-            maxWidth={350}
-            fontSize="13px"
-            my={2}
+      <Flex flexWrap="wrap" flexDirection="column" width="100%">
+        <Box color="black.800" fontSize="16px" fontWeight={700} lineHeight="24px" pt="26px">
```

Comment:
Would have been nice to use `Label` + `htmlFor` here for a more accessible form

Reply:
Done at, https://github.com/opencollective/opencollective-frontend/pull/8057